src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java

```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
 		}
 
 		// Cooking for Blockheads II
-		if(CookingConfig.enableCraftingBook) {
-			GameRegistry.addRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', Items.diamond, 'B', CookingForBlockheads.itemRecipeBook);
+		if (CookingConfig.enableCraftingBook) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', "plateSteel", 'B', CookingForBlockheads.itemRecipeBook));
+			} else {
+				GameRegistry.addRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', Items.diamond, 'B', CookingForBlockheads.itemRecipeBook);
+			}
 		}
 
 		// Fridge
-		GameRegistry.addShapelessRecipe(new ItemStack(CookingForBlockheads.blockFridge), Blocks.chest, Items.iron_door);
+		if (Loader.isModLoaded("gregtech")) {
+			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockFridge), "PSP", "PCP", "PSP", 'S', "screwIron", 'P', "plateIron", 'C', Blocks.chest));
+		} else {
+			GameRegistry.addShapelessRecipe(new ItemStack(CookingForBlockheads.blockFridge), Blocks.chest, Items.iron_door);
+		}
 
 		// Sink
-		if(CookingConfig.enableSink) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "ingotIron", 'W', "logWood", 'B', Items.water_bucket));
+		if (CookingConfig.enableSink) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "plateIron", 'W', "plateWood", 'B', Items.water_bucket));
```

Comment:
Does this plate need a compressor? If so, that pushes it to steam. I thought it was supposed to be pre-steam?

Same for other platewoods.

Reply:
I'm fine with steam.  Iron/wood/steel plates are fine.  I think it was already removed, but I'd avoid Diamond plates or anyting.